src/EventStore.php

```diff
@@ -388,4 +352,108 @@ public function getActionEventEmitter(): ActionEventEmitter
     {
         return $this->actionEventEmitter;
     }
+
+    private function preLoad(string $action, StreamName $streamName, int $fromNumber, int $count = null)
```

Comment:
It returns Stream|array. That's why there is no return type here. Also it's a private method, so nobody has to deal with it.